CONTRIBUTING.md

```diff
@@ -1,46 +1,56 @@
-# Contributing to OpenEBS / Maya
+# Contributing to OpenEBS Maya
 
-Are you ready to enhance Maya. Great!!!
+OpenEBS uses the standard GitHub pull requests process to review and accept contributions.  There are several areas that could use your help. For starters, you could help with improving the sections is this document by either creating a new issue describing the improvement or submitting a pull request to this repository. The issues for the various openebs components (including maya components) are maintained in [openebs/openebs](https://github.com/openebs/openebs/issues) repository.
 
-Feel free to contribute by:
-- raising issues
-- participate in commenting/voting on the issues
-- create new PRs for Documentation or Code
+* If you are a first time contributor, please see the below [Steps to Contribute](#steps-to-contribute)
```

Comment:
see the below = see Steps to Contribute.

Reply:
done

Reply:
first time - first-time